packages/Uploader/src/components/File/File.js

```diff
@@ -70,12 +69,14 @@ function File(props) {
         return (
           <Popover isDark isEager>
             <Popover.Tip />
-            <Popover.Trigger>
+            <Popover.Trigger tabIndex={-1}>
               <Button.Icon
+                aria-label={`${name} ${I18n.t("uploader.cancel_upload")}`}
```

Comment:
it's better to translate the entire sentence, e.g `aria-label={`${I18n.t("uploader.cancel_upload", {name})}`}` and in your yml file, `cancel_upload: Cancel upload {{name}}.` because sometime the words' order can change in other languages